.github/workflows/ci.yml

```diff
@@ -164,41 +164,39 @@ jobs:
           GITGUARDIAN_API_KEY: ${{ secrets.GITGUARDIAN_API_KEY }}
           GITGUARDIAN_API_URL: ${{ secrets.GITGUARDIAN_API_URL }}
 
-  dockerhub-unstable:
-    name: Push Docker image to Docker Hub
+  dockerhub-github_packages-unstable:
+    name: Push Docker image to Docker Hub and GitHub Packages
```

Comment:
nit: maybe shorten the job name to something like `publish_docker_images`? Long names are cropped in the sidebar:

<img width="706" height="162" alt="image" src="https://github.com/user-attachments/assets/3c7c404c-f230-48f1-947d-6b0b87088eb2" />